Test is flaky or the build is broken

Looks like it might be a timezone issue. See the log below:

```
 go test ./...
--- FAIL: TestDecodeColumn (0.00s)
    --- FAIL: TestDecodeColumn/date (0.00s)
        decoder_test.go:282: DecodeColumn(2018-01-22) = "DATE \"2018-01-22\"", want = "DATE \"2018-01-23\""
    --- FAIL: TestDecodeColumn/array_date (0.00s)
        decoder_test.go:282: DecodeColumn([2018-01-22 2018-01-23]) = "[DATE \"2018-01-22\", DATE \"2018-01-23\"]", want = "[DATE \"2018-01-23\", DATE \"2018-01-24\"]"
FAIL
FAIL	github.com/cloudspannerecosystem/spanner-dump	0.191s
FAIL
```

--- a/decoder_test.go
+++ b/decoder_test.go
@@ -69,6 +69,15 @@ func equalStringSlice(a []string, b []string) bool {
 	return true
 }
 
+func mustTimeParse(t *testing.T, timeStr string) time.Time {
+	t.Helper()
+	parsed, err := time.Parse(time.RFC3339, timeStr)
+	if err != nil {
+		t.Fatalf("time.Parse unexpectedly failed: %v", err)
+	}
+	return parsed
+}
+
 func TestDecodeColumn(t *testing.T) {
 	for _, tt := range []struct {
 		desc  string
@@ -153,7 +162,7 @@ func TestDecodeColumn(t *testing.T) {
 		},
 		{
 			desc:  "date",
-			value: civil.DateOf(time.Unix(1516676400, 0)),
+			value: civil.DateOf(mustTimeParse(t, "2018-01-23T05:00:00+09:00")),
 			want:  `DATE "2018-01-23"`,
 		},
 
@@ -232,7 +241,7 @@ func TestDecodeColumn(t *testing.T) {
 		},
 		{
 			desc:  "array date",
-			value: []civil.Date{civil.DateOf(time.Unix(1516676400, 0)), civil.DateOf(time.Unix(1516762800, 0))},
+			value: []civil.Date{civil.DateOf(mustTimeParse(t, "2018-01-23T05:00:00+09:00")), civil.DateOf(mustTimeParse(t, "2018-01-24T05:00:00+09:00"))},
 			want:  `[DATE "2018-01-23", DATE "2018-01-24"]`,
 		},
 

--- a/decoder_test.go
+++ b/decoder_test.go
@@ -69,7 +69,7 @@ func equalStringSlice(a []string, b []string) bool {
 	return true
 }
 
-func mustTimeParse(t *testing.T, timeStr string) time.Time {
+func mustParseTimeString(t *testing.T, timeStr string) time.Time {
 	t.Helper()
 	parsed, err := time.Parse(time.RFC3339, timeStr)
 	if err != nil {
@@ -162,7 +162,7 @@ func TestDecodeColumn(t *testing.T) {
 		},
 		{
 			desc:  "date",
-			value: civil.DateOf(mustTimeParse(t, "2018-01-23T05:00:00+09:00")),
+			value: civil.DateOf(mustParseTimeString(t, "2018-01-23T05:00:00+09:00")),
 			want:  `DATE "2018-01-23"`,
 		},
 
@@ -241,7 +241,7 @@ func TestDecodeColumn(t *testing.T) {
 		},
 		{
 			desc:  "array date",
-			value: []civil.Date{civil.DateOf(mustTimeParse(t, "2018-01-23T05:00:00+09:00")), civil.DateOf(mustTimeParse(t, "2018-01-24T05:00:00+09:00"))},
+			value: []civil.Date{civil.DateOf(mustParseTimeString(t, "2018-01-23T05:00:00+09:00")), civil.DateOf(mustParseTimeString(t, "2018-01-24T05:00:00+09:00"))},
 			want:  `[DATE "2018-01-23", DATE "2018-01-24"]`,
 		},
 

